dataactbroker/handlers/fileHandler.py

```diff
@@ -133,6 +133,28 @@ def getErrorReportURLsForSubmission(self, isWarning = False):
             # Unexpected exception, this is a 500 server error
             return JsonResponse.error(e,StatusCode.INTERNAL_ERROR)
 
+    def getSignedURLForSubmissionFile(self):
+        """ Gets the signed URL for the specified file """
+        try:
+            self.s3manager = s3UrlHandler()
```

Comment:
cm and I had this conversation as well and I found that in one of the other functions there's a line that says `self.s3manager = s3UrlHandler(CONFIG_BROKER["aws_bucket"])` which may or may not override the original vanilla declaration so I just set it again just in case to clear it out. I'm not sure if that's ever something that we'd run into but I wanted to make sure in case we did.